### PR TITLE
[WIP] Starting again with issue 34

### PR DESCRIPTION
**Pull request by hiromipaw**
- [x] Make resource classes available in ``cliquet/__init__.py`` to ease usage
- [x] Remove Base* prefixes from almost every cliquet classes

### CODE REVIEW

File: dump.rdb [hunk 1]
```diff
@@ -0,0 +1 @@
+REDIS0006�ܳC�Z��V
```
Comment (Natim) [inline dump.rdb]: We should not add this file.

Comment (hiromipaw) [inline dump.rdb]: Yep I will exclude it. :+1:

File: cliquet/__init__.py [hunk 2]
```diff
@@ -16,6 +16,7 @@
 from cliquet.initialization import (  # NOQA
     initialize, initialize_cliquet, install_middlewares)
 
+__all__ = ['utils', 'errors', 'resource']
```
Comment (Natim) [inline cliquet/__init__.py]: I am not sure of what you are trying to do here.

File: cliquet/resource.py [hunk 3]
```diff
@@ -12,7 +12,7 @@
 from cliquet import Service
 from cliquet.collection import Collection
 from cliquet.errors import http_error, raise_invalid, ERRORS
-from cliquet.schema import ResourceSchema, PermissionsSchema
+from cliquet.schema import Schema, PermissionsSchema
```
Comment (Natim) [inline cliquet/resource.py]: I think we wanted to remove Base but not Resource in #34

File: cliquet/__init__.py (hunk 2 above)
Comment (Natim) [inline cliquet/__init__.py]: If the aim is to do ``from cliquet import ResourceSchema`` then we should have a list of all the think we want to expose in ``cliquet``

Something like: ``__all__ = ['ResourceSchema', 'PermissionsSchema', 'register', 'Service', 'DEFAULT_SETTINGS', ...]``

Comment (Natim) [inline cliquet/__init__.py]: The exhaustive list of everything we want to expose is the following:

- ProtectedResource
- BaseResource (Renamed Resource)
- register
- ProtectedViewSet
- ViewSet

**Comment by Natim** [comment]
Just push new commits to your ``imports_refactoring`` branch to update your PR, no need to create a new PR.

File: cliquet/resource.py (hunk 3 above)
Comment (leplatrem) [inline cliquet/resource.py]: I was wondering that too... But maybe it's ok to just have ``cliquet.Schema`` and ``cliquet.Resource``.

**Comment by leplatrem** [comment]
Thanks Silvia !

File: cliquet/resource.py (hunk 3 above)
Comment (ametaireau) [inline cliquet/resource.py]: Having a `Schema` like that doesn't tell what it is.

**Comment by hiromipaw** [comment]
Hey guys, sorry I felt behind with this again...

**Comment by Natim** [comment]
Superseeded by #34

**Comment by Natim** [comment]
Thanks @hiromipaw I am merging this after a rebase.

**Pull request closed without merging**